richie-south/ultimate_pi
# Feature: Add User Hint for High/Low Guess in Number Game

### Description
This Pull Request adds a new feature to the existing codebase. Users can now receive hints regarding whether their guessed number is higher or lower than the target number in the number guessing game. This feature enhances the user experience by providing additional guidance.

### Changes Made
- Added _checkGuessLessOrMore function 
- Changes in package version

### How to Test
1. Navigate to go
2. Enter a number guess in the input field.
3. Submit your guess.
4. Observe the feedback, which will now include hints on whether your guess is too high or too low.

### Checklist
- [x] The code compiles and runs without errors.
- [x] The feature has been tested and is working as expected.
- [x] Proper code comments and documentation have been added.
